examples/get_event_200.json

```diff
@@ -272,6 +272,34 @@
       "data": {
         "result": false
       }
+    },
+    "velocity": {
+      "data": {
+        "distinctIp": {
+          "intervals": {
+            "5m": 1,
+            "1h": 1,
+            "24h": 1
+          }
+        },
+        "distinctLinkedId": {
+          "intervals": null
```

Comment:
Any reason we want to show null here? Versus omitting it

Reply:
With the omitempty tags I added I think this entire entry would just be missing

Reply:
Fixed this to 
```"distinctLinkedId": {}```